src/components/Icon/CrossIcon/CrossIcon.jsx

```diff
@@ -22,18 +25,26 @@ const CrossIcon = createClass({
 
 	propTypes: {
 		...Icon.propTypes,
+		large: bool`
```

Comment:
I think it makes more sense to call this `size` and make it an enum like we do [over here](https://github.com/appnexus/lucid/blob/f5a89a2033935b52cf1f79966072461d9beeb333/src/components/Button/Button.jsx#L60). Just make it a `oneOf(['large'])` and make the default 16x16.

Side note, our convention is to us isX and hasX for boolean props. Those conventions are mentioned in [our contrib docs](https://github.com/appnexus/lucid/blame/f5a89a2033935b52cf1f79966072461d9beeb333/CONTRIBUTING.md#L63).

Reply:
There's a problem with this because size is a prop that sometimes gets passed through to the generic Icon. What else could we call it?

Reply:
Oh dang, you're right. Good catch. How about `presetSize`?

Reply:
Sounds good to me